package.json

```diff
@@ -106,6 +106,7 @@
     "ora": "^4.0.3",
     "path-to-regexp": "^3.1.0",
     "polyfill-crypto.getrandomvalues": "1.0.0",
+    "querystringify": "^2.1.1",
```

Comment:
I'd always question the value of pulling in new deps as it adds to our bundle size. Is it not too much trouble to live without this package?

Reply:
I am using url-parse (which is already available in node) instead of querystringify